tools/cli.js

```diff
@@ -24,14 +24,64 @@ const log = (msg) => {
   if (userInput) rl.write(userInput);
 };
 
+function completeByFields(input, object) {
+  return Object.keys(object)
+    .filter(c => !c.startsWith('_') && (!input || c.startsWith(input)));
```

Comment:
@aqrln, but isn't it redundant? As far as I know, `String.prototype.startsWith` will return `false` when first argument is `undefined`.